OrbitClientData/FunctionUtils.cpp

```diff
@@ -17,7 +17,7 @@ uint64_t StringHash(const std::string& string) {
 }
 }  // namespace
 
-namespace FunctionUtils {
+namespace function_utils {
```

Comment:
So as discussed this namespace should be removed in favor of `orbit_client_data` and the functions contained renamed to include a hint to `FunctionInfo`. Should we do this at a later time?

Reply:
I suggest so. There were two FunctionUtils, one of which was removed by somebody while I was doing this which required a rebase for me. I was thinking alignment would be a first step, followed by removal later - especially since I don't yet have enough context to carry out the renaming in a satisfactory way.